R/run_pacta_reporting_process.R

```diff
@@ -0,0 +1,317 @@
+  peers_equity_results_user <- read_rds_or_return_alt_data(
+    filepath = file.path(benchmarks_dir, paste0(params[["project_code"]], "_peers_equity_results_portfolio_ind.rds")),
+    alt_return = pacta.portfolio.utils::empty_portfolio_results()
+  )
+
+  log_debug("Loading index bonds peer results.")
+  peers_bonds_results_user <- read_rds_or_return_alt_data(
+    filepath = file.path(benchmarks_dir, paste0(params[["project_code"]], "_peers_bonds_results_portfolio_ind.rds")),
+    alt_return = pacta.portfolio.utils::empty_portfolio_results()
+  )
+
+  log_debug("Loading index equity portfolio results.")
+  indices_equity_results_portfolio <- readRDS(file.path(benchmarks_dir, "Indices_equity_results_portfolio.rds"))
+
+  log_debug("Loading index bonds portfolio results.")
+  indices_bonds_results_portfolio <- readRDS(file.path(benchmarks_dir, "Indices_bonds_results_portfolio.rds"))
+
+  # create interactive report ----------------------------------------------------
+
+  prepare_interactive_report(
+    report_output_dir = report_output_dir,
+    project_report_name = params[["project_report_name"]],
+    survey_dir = survey_dir,
+    real_estate_dir = real_estate_dir,
+    language_select = params[["language_select"]],
+    peer_group = params[["peer_group"]],
+    investor_name = params[["investor_name"]],
+    portfolio_name = params[["portfolio_name"]],
+    start_year = params[["start_year"]],
+    currency_exchange_value = params[["currency_exchange_value"]],
+    select_scenario = params[["select_scenario"]],
+    scenario_other = params[["scenario_other"]],
+    portfolio_allocation_method = params[["portfolio_allocation_method"]],
+    scenario_geography = params[["scenario_geography"]],
+    sector_list = params[["sector_list"]],
+    green_techs = params[["green_techs"]],
+    tech_roadmap_sectors = params[["tech_roadmap_sectors"]],
+    pacta_sectors_not_analysed = params[["pacta_sectors_not_analysed"]],
+    display_currency = params[["display_currency"]],
+    audit_file = audit_file,
+    emissions = emissions,
+    portfolio_overview = portfolio_overview,
+    equity_results_portfolio = equity_results_portfolio,
+    bonds_results_portfolio = bonds_results_portfolio,
+    equity_results_company = equity_results_company,
+    bonds_results_company = bonds_results_company,
+    equity_results_map = equity_results_map,
+    bonds_results_map = bonds_results_map,
+    indices_bonds_results_portfolio = indices_bonds_results_portfolio,
+    indices_equity_results_portfolio = indices_equity_results_portfolio,
+    peers_bonds_results_portfolio = peers_bonds_results_portfolio,
+    peers_bonds_results_user = peers_bonds_results_user,
+    peers_equity_results_portfolio = peers_equity_results_portfolio,
+    peers_equity_results_user = peers_equity_results_user,
+    analysis_output_manifest = analysis_output_manifest
+  )
+
+  # create executive summary -----------------------------------------------------
+  prepare_executive_summary(
```

Comment:
nit: shouldn't executive summary be an optional process?

Reply:
It will be as part of `workflow.pacta.webapp`